File: pgm/ABC_Java/ABC_276_A.java
import java.io.*;
// import java.math.*;
// import java.util.*;
// import java.util.Map.*;
// import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final char a = 'a';
        final String S = reader.readLine();
        if (S.lastIndexOf (a) < 0) {
            System.out.println (S.lastIndexOf (a));
        } else {
            System.out.println (S.lastIndexOf (a) + 1);
        }
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_276_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.*
// import java.util.Scanner
// import java.util.StringTokenizer

// import kotlin.math.*

const val a:String = "a"
const val one: Int = 1

fun main() {
    val S: String = readln()
    val idx: Int = S.lastIndexOf(a)
    if (idx == -one) {
        println(-one)
    } else {
        println(idx + one)
    }
    kotlin.system.exitProcess(0)
}